.gitlab-ci.yml

```diff
@@ -121,7 +121,7 @@ build_windows:
     - cbindgen . -c cbindgen.toml | grep -v \#include | uniq >target/${CI_PROJECT_NAME}.h
     # python build
     - pip install -r src/interfaces/pyo3/tests/requirements.txt
-    - maturin build --release --features python --target x86_64-pc-windows-gnu
+    - maturin build --release --features python --target x86_64-pc-windows-gnu --target-dir target_python
   artifacts:
     paths:
       - target/x86_64-pc-windows-gnu/release/*.dll
```

Comment:
we should keep as artifact the `target_python` folder

Reply:
actually we don't need the libs from `./target_python`, the resulting .whl package is saved in `./target`